src/api/dashboard.ts

```diff
@@ -50,3 +68,30 @@ export async function postDashboardsInvitations(
     }
   }
 }
+
+export async function putDashboard(
+  dashboardId: string,
+  title: string,
+  color: string,
+) {
+  const data = {
+    title: title,
+    color: color,
+  };
```

Comment:
nit
key값과 value값이 같아서 이렇게 해도 될 것 같아요!

```suggestion
  const data = {
    title,
    color,
  };
```

Reply:
nit 감사해요!! 👍👍👍